src/pages/New.js

```diff
@@ -25,20 +25,23 @@ export default function New() {
     postRestaurant({
       imgSrc:
         "https://static.lieferando.de/images/restaurants/de/OPROO0P/logo_465x320.png",
-      title: "Pasta Pronto",
+      title: input,
       categories: ["pizza", "salad", "pasta"],
       distance: 6,
       rating: 3.1,
       description: "Eat italian"
     });
   }
-
+  const [input, setTitleInput] = useState();
```

Comment:
better use `title` because it represents the title of the restaurant.
The set state function should be named like `setTitle`.

And I would set a default state like an empty string.